### PR TITLE
docs(aio): change hardcoded links to the old site

### PR DESCRIPTION
**Pull request by Foxandxss**
While this change is pretty much straightforward, I require @wardbell since I made a prose change.

### CODE REVIEW

File: aio/content/guide/aot-compiler.md [hunk 1]
```diff
@@ -727,15 +727,11 @@ Rollup does the tree shaking as before.
...
-The general audience instructions for running the AOT build of the Tour of Heroes app are not ready.
-
-The following instructions presuppose that you have cloned the
-<a href="https://github.com/angular/angular.io" target="_blank">angular.io</a>
-github repository and prepared it for development as explained in the repo's README.md.
-
-The _Tour of Heroes_ source code is in the `public/docs/_examples/toh-6/ts` folder.
+  The following instructions presuppose that you have downloaded the
+  <a href="/content/zips/toh-pt6/toh-pt6.zip" target="_blank">Tour of Heroes' zip</a>
+  and ran `npm install` on it.
```
Comment (wardbell) [inline aio/content/guide/aot-compiler.md]: "run"

File: aio/content/guide/setup.md [hunk 2]
```diff
@@ -364,7 +364,7 @@ As much fun as this is ...
 Use the <live-example title="QuickStart Seed in Plunker">live coding</live-example> environment as a _playground_, 
 a place to try the documentation samples and experiment on your own.
 It's the perfect place to reproduce a bug when you want to
-<a href="https://github.com/angular/angular.io/issues/new" target="_blank" title="File a documentation issue">file a documentation issue</a> or
+<a href="https://github.com/angular/angular/issues/new" target="_blank" title="File a documentation issue">file a documentation issue</a> or
```
Comment (wardbell) [inline aio/content/guide/setup.md]: Igor wants us to remove `target="_blank"`

**Review by wardbell — APPROVED** [review]
LGTM.  One word change ... "run" replaces "ran" ... and its good. I trust you'll make that change and so I "approve".

File: aio/content/guide/aot-compiler.md (hunk 1 above)
Comment (Foxandxss) [inline aio/content/guide/aot-compiler.md]: Right

**Comment by Foxandxss** [comment]
This one is good to go

**Pull request merged**